Makefile

```diff
@@ -124,10 +124,6 @@ build: clean build.common docker.ndm docker.ndo docker.exporter
 .PHONY: build.common
 build.common: license-check-go version
 
-# Tools required for different make targets or for development purposes
-EXTERNAL_TOOLS=\
-	github.com/mitchellh/gox
-
 # Bootstrap the build by downloading additional tools
 .PHONY: bootstrap
 bootstrap:
```

Comment:
nitpick: The recipe lines inside bootstrap can also be commented out (can keep the `bootstrap` target, ). Since the `EXTERNAL_TOOLS` variable is no more present. 
Also, give a comment, if any tools need to be added, define it in the variable.